dotnet/tests/MicroFocus.CafApi.QueryBuilder.Vqll.Parser.SystemJson.Tests/SystemJsonFilterReaderTest.cs

```diff
@@ -0,0 +1,825 @@
+/**
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xunit;
+
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using MicroFocus.CafApi.QueryBuilder.Vqll.Builders.SystemJson;
+using Microsoft.Extensions.Logging;
+
+namespace MicroFocus.CafApi.QueryBuilder.Vqll.Parser.SystemJson.Tests
+{
+    public sealed class SystemJsonFilterReaderTest
+    {
+        private static readonly ILogger<SystemJsonFilterReaderTest> _logger =
+            LoggerFactory.Create(b => b.AddDebug().AddConsole())
+            .CreateLogger<SystemJsonFilterReaderTest>();
+
+        private static readonly ILogger<SystemJsonFilterReader> _filterReaderlogger =
+            LoggerFactory.Create(
+                b => b
+                .AddDebug()
+                .AddConsole()
+                .AddFilter("MicroFocus.CafApi.QueryBuilder.Vqll.Parser.SystemJson.SystemJsonFilterReader", LogLevel.Debug))
+            .CreateLogger<SystemJsonFilterReader>();
+
+        [Fact]
+        public void TestParseEquals()
+        {
+            string query = @"[""=="", ""TITLE"",""classified""]";
```

Comment:
Was using backticks everywhere else but not here intentional?

Reply:
Yes, the backticks make the strings more readable.

Reply:
Ok.  But that doesn't answer the question.

Reply:
This change was just part of my learning about verbatim strings in c#.